### PR TITLE
Removes science's ability to produce shotgun shells that instantly kill and delimb their target. If you guys get mad at this just wait until you see what I have planned for botany lmao

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Can you not fucking read the title?

Anyway it removes frag-12 rounds, which, for *literal corn from botany*, lets you fire gyrojet rounds. These things ignore armor, ignore everything, and just fucking delimb you. They're hilariously broken.

Meteorshot's gone too, because...I mean, breaching rounds exist to do the same thing. And meteorshot is infamously badly balanced.

These are only removed from crew craftable things. If you find them in the wild or add them to an antag, hey, have fun. You still have pulse slugs and tazer rounds, so you can use those instead.
#### Changelog

:cl:  
rscdel: Comments out frag-12 rounds and meteorshot. Not that it matters since buckshot kills anything in two hits.
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
taser slugs dont exist, PR is still great. I don't remember the last time i saw frag-12 rounds but meteor slugs are overpowered for something you can just CRAFT.

edit: or should i say, taser slugs arent craftable anymore

**Review by jcatjmeow — CHANGES_REQUESTED** [review]
Just delete them. We have github history if someone wants to add them back. No need to keep the comments

**Comment by Rdrazga** [comment]
Alternative of fixing the shell itself should be looked at since its not supposed to be a 1-shot and for some reason the code is making it one, If a fix is possible then add it to nukies to give it more use.

**Comment by boodaliboo** [comment]
> Just delete them. We have github history if someone wants to add them back. No need to keep the comments

your dumb its good to keep items even if unobtainable for future use like for example when  this pr gets merged ill add them to something if i remeb er

**Comment by Readystorm546** [comment]
i ded

**Comment by Tipy1802** [comment]
How is this a problem? Science can’t even use said shells since they can’t produce guns

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Delete instead of commenting

**Comment by ghost** [comment]
@Tipy1802 they can make improvised shotguns, and besides, the issue isn't that it can or can't be used, just that it exists

**Comment by Georgeur** [comment]
I'm against this. The idea that you can just get glycerol from botany like its just nothing, isn't a nuanced opinion. Meteorshot isn't even far off being balanced, its the stun that pisses people off, it's literally the same problem people have with the meathook. There also hasn't been adequate rounds in which these items were used to make this all that necessary, and that's coming from the only guy I know of, openly shouting about the zombie powder nerf being very necessary. My largest concern is however is that removing these items wholesale seems excessive, and it is excessive. Also nothing to do with the actual PR, as its just an ad hominem, but Adam clearly either has no idea on the state of balance on the server, had no idea taser slugs were removed, and I've heard hasn't even played the server in a very long time.

**Comment by ghost** [comment]
@Georgeur 

It's fine that you're against that, it just means you're stupid.

Glycerol can be gotten from the public garden. The fact that something isn't used doesn't mean it won't be used. And when this IS used, its only purpose is to instantly kill someone.

Seriously. That's all it does. It instantly kills someone. This is its only purpose. It isn't like meth, or xenoling, where it's OP but can be used in ways that are not OP. This slug's literal only purpose is to instantly kill someone and it's better than everything else in the game.

**Comment by Georgeur** [comment]
After testing. The frag 12 rounds do not in fact, ignore armor (either from the bullet or explosion), ignore 'everything', or delimb you (took on average 12 shells total for a delimb, and yes, they were also already very dead, its a 0/0/1 explosion of course it might delimb a corpse). Is there any reason you wanna remove them, that's actually true?
The only other arguments left are to do with meteorshot, and I'd say it functions a different role from breaching rounds. The claim its "infamously badly balanced" isn't one that can be proved, but more importantly still just an appeal to the crowd, a logical fallacy. 
TLDR: Every single one of your arguments is either veritably false, or is logically flawed. Please correct these issues.

**Comment by Georgeur** [comment]
Adam. To make a Frag 12 shell, you also need, fluorosulphuric acid and sulphuric acid, might I ask if I am stupid. Does this not mean you'd need a chem dispenser. I'll save you the trouble. I already made the list, and yes, this is purposely excluding roundstart flurosulphuric acid spawns.
you need sulphuric acid
no way to get that with ghetto chem
fluorine 
only one way 0.5u per water bottle....
potassium
only one way really, tear gas grenades...
and hydrogen
10u per glowstick

Edit: If you're ganna say deathnettles, and then farm bananas, I dont get how the whole thing isnt a waste of time, especially in ghetto botany. It's so ineffective. You might as well be a normal botanist. Then, is this really a science problem? It'd help if most of your PR wasn't explicitly trying to be offensive, and actually made sound arguments for your vision.

**Comment by Reed0506** [comment]
he said science because they have the ability to print everything they need for them and just go grow some corn.

this PR is good.

I have personally tested FRAG-12s.
delimb almost guaranteed
average 100 damage per hit
stun that is not short

The only reason you guys are complaining about this shit is because you have not been on the other end of a FRAG-12.
I guarantee that if you were ever shot with them in normal gameplay you would think they were fucking overpowered and must be some mega TC traitor item.
Except they are effectively FREE and can be made by literally anyone who wants to make them and knows how.

Saying FRAG-12s are fine is like saying its okay for the science man to have what is effectively a combat mech missile rack in his backpack.

**Comment by Georgeur** [comment]
As it turns out, after multiple tests across many clients. For some reason, one instanced client, had no issues. Across about 4 other clients. This (frag 12 slugs) instantly puts anyone into 80/90 damage. This is a bigger bug fix issue than anything. I'm happy to do a 180, this didn't used to do this. Holy shit, upvote until fix.

**Comment by Georgeur** [comment]
I'm sorry, did you just ignore everything I said about chemistry. It's not just, let's just grow some corn, because the argument was 'just grow some corn' I must ask, are you mentally retarded.

**Comment by redmoogle** [comment]
> 
> 
> Adam. To make a Frag 12 shell, you also need, fluorosulphuric acid and sulphuric acid, might I ask if I am stupid. Does this not mean you'd need a chem dispenser. I'll save you the trouble. I already made the list, and yes, this is purposely excluding roundstart flurosulphuric acid spawns.
> you need sulphuric acid
> no way to get that with ghetto chem
> fluorine
> only one way 0.5u per water bottle....
> potassium
> only one way really, tear gas grenades...
> and hydrogen
> 10u per glowstick
> 
> Edit: If you're ganna say deathnettles, and then farm bananas, I dont get how the whole thing isnt a waste of time, especially in ghetto botany. It's so ineffective. You might as well be a normal botanist. Then, is this really a science problem? It'd help if most of your PR wasn't explicitly trying to be offensive, and actually made sound arguments for your vision.

problem is that you can easily get a chem dispenser from science or medical(especially them) as they either dont care or are loose on access requirements. or if your planning to murderbone or something you can already break in at mach 40 most likely

**Comment by ghost** [comment]
@Georgeur

Good, you can actually think.

Yes, you can get it from deathnettles. Or you can ask chemistry for it. It really, _really_ is not hard to get it from chemistry-either get a chem board yourself from the medlathe, which you will have access to as you are a scientist, or you can go into chemistry and make it yourself. You _will_ get in, especially if you're carrying an RPED, or you can just ask the AI to let you in, or ask a chemist. All three of those work.

The acid is not hard to get. It takes about one minute, if that. The hardest part and the constraining part is the corn oil, which can be grown in science with literally just the corn seeds from the public garden, and everyone knows how trivially easy it is to produce massive amounts of product.

So, as a result, you can get quite a good amount of Frag-12 shells. Getting a hydroponics tray is as simple as either using the public garden, printing the board, or just grabbing one from botany after getting the AI to let you in by wrenching the tray and leaving with it. The botanists won't care. No botanist EVER uses all their trays.

So yes, you do need a chem dispenser. This is not anywhere near the obstacle that you claim it is.

Frag-12 is gated behind a resource requirement that is, frankly, barely one. It isn't something you can just print in science outright, but you can get the requisite materials within a few minutes. Corn and acid. That's all it takes. These are not difficult or rare materials to get your hands on. They are not locked behind time, or behind something that makes it impossible to get in your average round, like an anomaly core, et cetera. It merely needs a *very simple* chemical set from chemistry. Not a complex one, like skewium. Acid and facid. That's all it takes.


Now. Let's examine what it can do. This shotgun shell will outright kill anything it hits, no matter what, no matter what it is or what it's wearing. 

Compared to shotgun slugs (60 damage), revolver shots (60 damage), pulse rifle shots (50 dmg, 40 if it's the shotgun variant, destroys walls), laser shots (20 damage)...you can see that this is out of whack.

In fact, the only comparable item that one-shot kills is the nukie RPG, which is...a nukie weapon. And, unlike shotguns, it can only hold one rocket per shot. In fact, another comparable item would be the nukie M-90gl, which also has a limited number of shots for firing explosive rounds. Oh, and the nukie syndicate cyborg grenade launcher, which fires similar explosive rounds and is not rechargable.

Do you see the consistency here? All weapons that fire explosive rounds are either difficult to or straight up not reloadable, and have a limited number of shots, usually one or two, before they need to be reloaded. The exception being the syndicate cyborg, which has six, but is not reloadable-and, oh yeah, they're ALL NUKIE WEAPONS.

Meanwhile, science can make 20 of these things from 100u of each requisite chemical. Given that the only REAL limiting factor is glycerol, and that glycerol is not hard to mass produce, these can be reliably be mass produced in extremely large numbers. 

And each one of them ends you in one shot.

That's why i'm cutting them out of crew production, Georgeur. Does that make sense to you, or do I have to simplify it for you even further?

**Comment by Reed0506** [comment]
georgeur, sorry for not specifically explaining, but SCIENCE CAN PRINT THEIR OWN CHEM SETUP WITH EASE.

Also the cause of all issues with the FRAG-12 is the -1,0,1 explosion that it creates on impact. I have personally tested FRAG-12s multiple times and the explosion is what delimbs, deals heavy damage, and stuns. Explosion light impact is nothing to sneeze at when it comes to damaging people.

**Comment by Georgeur** [comment]
Upon further testing, frag12 shells are less powerful than you say they are, but still far too powerful to be kept in the game in their current state. Without armor, damage ranges roughly from 90-120 with typically one or sometimes two delimbs, with a full bomb suit damage is consistently 55 with no delimbs. 
I do think wholesale removing meteorslugs is a bad idea, they could be easily balanced by changing their paralyse from 50, down to something like 5, hell even 1. Maybe they should be seen as a type of heavy duty beanbag, knocking down your opponent but not stunning them. This should be also viewed in the eyes of that armor reduces stun times. 
Edit: It appears that is simply the base damage of a 0/0/1 explosion, which when spawned on a players tile, has a 50/50 chance of dealing 60 damage, or within the previously mentioned range of damage.

**Comment by ghost** [comment]
@Georgeur 

beanbags are for nonlethal stuns. breaching shells are for airlock removal. meteorshot is just overpowered, in part because it throws your opponent, so if they hit a wall they're stunned for longer and take extra dmg

**Comment by alexkar598** [comment]
So from my testing, the meteorslug is a tad too similar to tazers to remain in its current state but theres really nothing wrong with the frag 12, it stuns on the first hit but if you just take normal shotgun slugs, it takes 2 hits to crit so theres really nothing much to upset the balance here. It does give you a slight advantage over the more easily mass produced slugs so the slight buff it gives you is justified as far as I can tell. The meteorshot was probably more balanced around when tazers were a thing and could use a nerf on their stun which is a death sentence combined with a high power melee item.

**Comment by ghost** [comment]
@Alek2ander They crit in one hit and blow off limbs. They do this even through shielded hardsuits.

That's not okay, Alek. Unless you're okay with science being given printable RPGs.

**Comment by Georgeur** [comment]
@Alek2ander I'm not sure why, but on some peoples instances frag12 doesn't seem to act in the way it does on other clients. Adam is right in that, it deals to an unclothed person, about 90-120 damage on average, on my instance. They're wrong about it ignoring armor and always critting either way though, due to how limb damage effects total damage, you might think it's a crit, but 9/10 times its not a crit (its still a metric shit tonne of damage), and with bomb armor, tested over 100 times, ive always seen the same damage, with no delimb. As ever Adam is being hyperbolic and actually making it harder for me, and I'm sure others, to bloody help them. 
Edit: I can only say that from third party experiences, but, it was from a couple of guys I do believe, so I have no idea what's up with frag12 sometimes just causing instant delimb, or like 60 damage max often less, depending on the instance, or something I'm somehow completely ignorant of.

**Comment by Georgeur** [comment]
@AdamElTablawy 
damage is negligible, and yes the wallstun is annoying, its still not broken. Like legit, good find on the frag12 (even though I'm starting to think it's bugged as on a large minority of peoples private servers it doesnt seem to work the same, not discounting it, very good find), but why the hell do you think meteorslug should just be removed? 20 damage, plus maybe the stun is removed for balance, plus it knocks you back. I don't get it. Explain it to me.

**Comment by ghost** [comment]
@Georgeur I don't count bomb armor because nothing has fucking bomb armor except for EOD suits, which nobody wears, because they're worthless except in this one, very niche, situation of being shot by a frag-12 round.

They still flatten you regardless in one. In *actual* combat situations, such as a nukie in an elite hardsuit, he is going to fucking die.

As for the meteorshot, we removed ranged stuns. So we're removing this too. There's no real reason it SHOULD remain as a crew-craftable weapon. It's nukie gear by default. I don't favor giving the crew nukie gear. Instead of arguing why I shouldn't remove it, ask why it deserves to stay? What purpose does it serve that is not filled by either slugs or breaching rounds?

**Comment by Georgeur** [comment]
@AdamElTablawy many things have bomb armor, elite nukie armor has 55 of course, stop talking our your ass, they don't get delimbed, and like I've maintained, they definitely dont go into crit, like most the time people dont without ANY armor. I'm trying to support your argument, and you're still being a child about the situation. 

Now about meteorslug.
Yes and I mentioned getting rid of the stun for balance reasons.
Saying something is nukie gear 'by default' is not only kinda just your opinion, but saying anything is nukie gear by default is childish. What is the only thing that differentiates the combat shotgun and the bulldog the tiny difference in capacity, or is it also the manual loading of shells. Give an actual reason, instead of purposely being difficult. 
I'm all in favour about getting rid of the stun, but you want it gone, wholesale.

**Comment by ghost** [comment]
@Georgeur i'm nerfing combat shotguns anyway given they're hysterically overpowered but that's later when i can figure out how to nerf them without making them straight up just shittier than the riot shotgun, so the point of nukie bulldog/combat shotgun equivalency is moot (granted, of course, the bulldog is much much better than the combat shotgun being that you can load it with a higher ammo mag and swap mags out where the combat shottie takes individual shells, so i'm not even sure what argument you were aiming for there)

anyway no i mean it's not my opinion, meteorshot *is* literally, nukie gear. it's for bulldog shotguns. i don't want to nerf it for nuke ops, and i'm not inclined to arbitrarily making a weaker subtype for the station to use and the nukies getting the full monty, so i'd rather remove them from _station crafting_ while leaving the plausibility of finding them as ruin loot, et cetera, in their otherwise functional form.

lastly, as for the fucking bomb armor debacle

yes, i know things have bomb armor, yes, i know elite nukie hardsuits have 55 bomb armor, cap's carapace has 25 bomb armor et cetera et cetera

however i do not actually care because it's not important. you can stop trying to 'support' my argument if all you're going to do is try and fail to undermine it for no particular reason with irrelevant semantics of 'oh this person can just barely survive being shot with one frag-12 round, and not the consecutive six more that will follow'

for all intents and purposes anyone hit with a frag-12 round is going to fucking die. have we agreed on this? good. glad that's settled. bomb armor doesn't matter, exceptions that will rarely ever occur in the course of gameplay don't matter. anyone who's hit with this will die or lose a limb most of the time. and we agree that they need to go, so drop it.

**Comment by Marmio64** [comment]
georgeur why are you fighting for this so bad? The bullets don't contribute anything good and there is no reason to keep them. FRAG-12 rounds are OP, if you don't think so, you are indeed mentally handicapped. Meteor slug rounds are also OP and if you remove their stun they'll practically just be breaching slugs that toss objects and people around, this alternative doesn't sound too interesting to me but im not completely against it either.

And remember, just because something is hard to get, doesn't mean its balanced.

**Comment by JamieD1** [comment]
Lets give it a go

**Review by JamieD1 — APPROVED** [review]

**Comment by Georgeur** [comment]
@AdamElTablawy considering it's been merged, and I'm certain it will not be reverted, as this does solve more than it harms, I'll make this reply short for both our sakes.

The point about combat shotguns vs the bulldog is perfectly fair, and saying sometime in the ether somewhere you'll nerf it so that means right now there's a big difference is laughable. The point you missed is that having either one to two extra shells per full load, and being able to reload with a magazine rather than shells, is not a big enough difference to say just because something is gear available to nukies makes it overpowered.
tldr: something just being a traitor item doesn't make it good, or better than a non traitor item.

I'm in agreement about shotguns being overpowered, not just the combat shotgun, the riot shotgun is basically just as powerful as the combat shotgun, cause by spamming use item and firing (clicking) you wont eject unused shells and the rate of fire seems to be slightly less if not on par with the combat shotgun.

The second point is I think a misunderstanding of how I'm positioning myself in this argument. You attempt to aggravate, which isn't all that bad, but then you won't admit you're wrong or being inaccurate. How else can I try to get to the truth of the matter with someone so boldly arrogant or ignorant, without trying to make sure lies aren't spread and believed on a github pr. I've made concessions, not to you but the PR, cause you're of course not always wrong, believing you're always right has made any productive attempt to improve the PR challenging to say the least.
tldr: You talk out your ass and revert into acting like an arrogant asshat whenever someone disagrees with you.

**Comment by Georgeur** [comment]
@Marmio64 
I believe either of the headcoders who've read my comments in full, understand I believe this PR does more good than bad, and if they haven't read it they wouldn't care.
Anyone who has spent the time to read my comments, will have seen me say I think they're too powerful to even be in the game. As well as some ideas that their behaviour seems different across other peoples servers, and there might be a bug involved.
The idea of reducing the stun to almost instant, is that they'd still drop what they're holding, and be thrown around.
Finally something being hard to get is a scale, and should be most times in the proportion to the power of the item. Hence why you can get the meteor gun every round which does as much damage as a frag12 and can be recharged like an egun.
I've spent the majority of this PR being confused on the different effect it has in other peoples testing, sometimes not being powerful at all.
